extensions/amp-story/1.0/animation.js

```diff
@@ -193,8 +231,8 @@ class AnimationRunner {
     return this.keyframes_.then((keyframes) => ({
       keyframes,
       target: this.target_,
-      delay: `${this.delay_}ms`,
-      duration: `${this.duration_}ms`,
+      delay: this.delay_,
+      duration: this.duration_,
```

Comment:
(`amp-animation` specs take numbers for time values, in which case they're `ms`. We didn't need the string with the unit and this simplifies things somewhat.)